prowler/providers/aws/services/iam/iam_service.py

```diff
@@ -661,6 +665,27 @@ def __list_tags_for_resource__(self):
                 f"{self.region} -- {error.__class__.__name__}[{error.__traceback__.tb_lineno}]: {error}"
             )
 
+    def __get_last_accessed_services__(self):
+        for user in self.users:
+            details = self.client.generate_service_last_accessed_details(Arn=user.arn)
+            response = self.client.get_service_last_accessed_details(
+                JobId=details["JobId"]
+            )
+            while response["JobStatus"] == "IN_PROGRESS":
+                response = self.client.get_service_last_accessed_details(
+                    JobId=details["JobId"]
+                )
+            self.last_accessed_services.append(
+                {"user": user, "services": response["ServicesLastAccessed"]}
+            )
+
+    def __get_access_keys_metadata__(self):
```

Comment:
Please include two `try/except` blocks here, one above `for user in self.users:` and another below not to stop the iteration if something fails.